data/tlg0062/tlg003/tlg0062.tlg003.perseus-eng4.xml

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://epidoc.stoa.org/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="eng">Dionysus, an Introductory Lecture</title>
+<sponsor>Tufts University</sponsor>
```

Comment:
We would normally also include the author and editor/translator name up here as wel.

Reply:
fixed